keyboards/crkbd/keymaps/thunderbird2086/process_records.c

```diff
@@ -23,18 +24,30 @@ bool process_record_oled(uint16_t keycode, keyrecord_t *record) {
 #endif
 
 bool process_record_user(uint16_t keycode, keyrecord_t *record) {
-    switch (keycode) {
-
-    case COLEMAK:
-        if (record->event.pressed) {
-            set_single_persistent_default_layer(_COLEMAK);
-        }
-        break;
-    case QWERTY:
-        if (record->event.pressed) {
-            set_single_persistent_default_layer(_QWERTY);
+    if (record->event.pressed) {
+        switch (keycode) {
+            case COLEMAK:
+                set_single_persistent_default_layer(_COLEMAK);
+                break;
+            case QWERTY:
+                set_single_persistent_default_layer(_QWERTY);
+                break;
+#           if defined(RGBLIGHT_ENABLE) || defined(RGB_MATRIX_ENABLE)
+            case RGBRST:
+#               if defined(RGBLIGHT_ENABLE)
+                eeconfig_update_rgblight_default();
+                rgblight_enable();
+#               else 
+                eeconfig_update_rgb_matrix_default();
+#               endif
+                break;
+#           endif
+            case VRSN:
+                send_string("# qmk " QMK_VERSION "\n" SS_DELAY(VER_NEWLINE_WAIT));
+                send_string("# " QMK_KEYBOARD "/" QMK_KEYMAP "\n" SS_DELAY(VER_NEWLINE_WAIT));
+                send_string("# built on: " QMK_BUILDDATE "\n");
```

Comment:
Well, should be able to make it part of one command. 
 Eg: 
 ```suggestion
                send_string("# qmk " QMK_VERSION "\n" SS_DELAY(VER_NEWLINE_WAIT) \
                            "# " QMK_KEYBOARD "/" QMK_KEYMAP "\n" SS_DELAY(VER_NEWLINE_WAIT) \
                            "# built on: " QMK_BUILDDATE "\n");
```

Reply:
If that's the case, then I think I can remove the delay.